README.md

```diff
@@ -415,6 +476,9 @@ settingsBuilder.createClusterOperationSettings()
 ClusterControllerClient clusterControllerClient = ClusterControllerClient.create(settingsBuilder.build());
 ```
 
+Note: The configuration above *only* modifies the LRO RetrySettings for the `createClusterOperation` RPC.
```

Comment:
```suggestion
Note: The configuration above *only* modifies the LRO values for the `createClusterOperation` RPC.
```
I may missed this sentence and the one below yesterday, feel free to use a slightly different wording. The point is `RetrySetting` is an implementation detail that may confuse customers, I think we should avoid it as much as possible.